MapboxNavigation/NavigationViewController.swift

```diff
@@ -18,6 +18,13 @@ public protocol NavigationViewControllerDelegate: VisualInstructionDelegate {
      - parameter canceled: True if the user dismissed the navigation view controller by tapping the Cancel button; false if the navigation view controller dismissed by some other means.
      */
     @objc optional func navigationViewControllerDidDismiss(_ navigationViewController: NavigationViewController, byCanceling canceled: Bool)
+	
+    /**
+     Called when user arrived at the destination of the trip.
+	 
+     - parameter navigationViewController: The navigation view controller that finished navigation.
+     */
+    @objc optional func navigationViewControllerDidFinish(_ navigationViewController: NavigationViewController)
```

Comment:
```suggestion
    @objc optional func navigationViewControllerDidFinishRouting(_ navigationViewController: NavigationViewController)
```

Take it or leave it naming nit.